app/containers/message/PhotoModal.js

```diff
@@ -75,10 +116,11 @@ export default class PhotoModal extends React.PureComponent {
 						<ImageViewer
 							imageUrls={[{ url: encodeURI(image) }]}
 							onClick={onClose}
+							onSave={() => this.saveToCameraRoll(encodeURI(image))}
```

Comment:
`onSave={this.saveToCameraRoll}` and do `encodeURI` inside of `saveToCameraRoll`